WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -87,14 +87,18 @@ class GutenbergSettings {
 
     /// True if gutenberg editor has been enabled at least once on the given blog
     func wasGutenbergEnabledOnce(for blog: Blog) -> Bool {
-        return database.object(forKey: Key.enabledOnce(for: blog)) != nil
+        return database.object(forKey: Key.enabledOnce(for: blog)) != nil || database.object(forKey: Key.appWideEnabled) != nil
```

Comment:
I don't think we want this, it means that the dialog will show only once per user and not per site. If you migrate from 12.9 and create a new site, it's getting gutenberg for new posts but not showing the dialog.

Reply:
>  If you migrate from 12.9 and create a new site, it's getting gutenberg for new posts but not showing the dialog.

True! I wasn't aware that this was required, but I confirmed internally with @hypest 

I'll give the other option setting the flag on the migration itself a try.